entity-management-web/pom.xml

```diff
@@ -181,26 +192,6 @@
 		<finalName>${project.artifactId}</finalName>
 
 		<plugins>
-
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-resources-plugin</artifactId>
-				<configuration>
-					<!-- 
-					<outputDirectory>output-resources</outputDirectory>
-					 -->
-					<resources>
-						<resource>
-							<directory>src/main/resources</directory>
-							<includes>
-								<include>*.properties</include>
-							</includes>
-							<filtering>true</filtering>
```

Comment:
resource filtering needs to be activated in order to fill the build.properties with the values from the pom file.  Please revert back this deletion

Reply:
Everything in `src/main/resources` is included in the classpath by default so the filtering is unnecessary. 

It also breaks the application – as everything apart from `.properties` files are excluded from the build. This isn't what we want.